db/db.go

```diff
@@ -265,7 +265,13 @@ func IsConnectionError(err error) bool {
 	if err == nil {
 		return false
 	}
-	lowerCaseError := strings.ToLower(err.Error())
+
+	errMsg := err.Error()
+	if cmdErr, ok := err.(mongo.CommandError); !ok {
```

Comment:
shouldn't this be `ok`, rather than `!ok`?

Reply:
whoops - forgot to change.